include/boost/math/tr1.hpp

```diff
@@ -106,8 +106,12 @@ namespace boost{ namespace math{ namespace tr1{ extern "C"{
 #  endif
 #  include <boost/config/auto_link.hpp>
 #endif
-#else
-#error Auto linking not supported in standalone mode.
+#else // Standalone mode
+#  ifdef _MSC_VER
```

Comment:
I don't think we need this - auto_linking is msvc only, and the user can still use the code without it, they just need to manually link.  At most a warning for msvc would be enough I think?